apps/central-scan/backend/src/app.ts

```diff
@@ -207,6 +208,52 @@ function buildApi({
       }
     },
 
+    getNextReviewSheet() {
```

Comment:
Could you add return type annotations to these API methods? It's a convention that I think is helpful